docs/user-guide/home/global-modules/README.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_navigation:
+  title: Global modules
+  priority: 999
+description: Global modules in OpenProject
+keywords: global modules, project overarching modules, global index pages
+---
+# Global modules
+
+Global modules present an overview of the projects that you are a member of or have permissions to see. Here you will find a summary of all the entries in the respective modules across your projects, such as **Activity, Work packages, Calendars, Boards**, etc.
+
+To find **Global modules** menu simply click on the logo in the header of the application, it will be displayed on the left side. 
+
+![Navigating to global modules menu in OpenProject](open_project_user_guide_global_modules_menu.png)
+
+> It is possible that some of the global modules are not displayed for you. This will be the case if said module is not activated in the [Project settings](../../projects/project-settings/) in any of the active projects in you instance. If for example **News** module is not enabled for any of the active projects, you will not see it in the **Global modules** menu. 
+
+## Projects
+
+**Projects** global module will display a list of the projects that exist in your OpenProject instance, which you are a member of and/or have the right to see (for example as an administrator) and public projects.
+
+You can create a new project here directly by using the  **+ Project** button at the bottom of the left side menu.
+
+![OpenProject projects overview in the global modules menu](openproject_global_modules_projects.png)
+
+On the left you will have the following options:
+
+**All projects** is the default view and shows only active projects. If you do not have administrator rights, this view will be the same as **My projects** for you.
```

Comment:
The part of administration rights and being the same as My projects is not true. This menu entry will contain both projects that you are member of and Public projects independent of your rights on them.

Reply:
It is true though that only active projects are displayed and archived projects are not.